tests/test_annotations.py

```diff
@@ -72,14 +72,17 @@ def test_merge_df(self):
         )
         cell_metadata_df.preprocess()
         cell_names_cell_metadata_df = np.asarray(cell_metadata_df.file['NAME'])
+        print(cell_names_cell_metadata_df)
```

Comment:
This print statement (and the next three print statements) look like they may be for debugging?

Reply:
553dd81